LiteCore/Database/TreeDocument.cc

```diff
@@ -764,5 +746,22 @@ namespace litecore {
         return asInternal(collection())->keyStore().withDocBodies(docIDs, callback);
     }
 
+    /*static*/ revidBuffer DocumentFactory::generateDocRevID(slice body,
```

Comment:
This should be a method of TreeDocumentFactory since it's specific to rev-trees.